ajax/libs/qcobjects/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "qcobjects",
+	"description": "QCObjects a Quickcorp Objects Library - Cross Browser Javascript Framework for MVC Patterns",
+	"filename": "QCObjects.js",
+	"license": "LGPL-3.0",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/QuickGroup/QCObjects.git"
+	},
+	"keywords": [
+		"node",
+		"quickcorp",
+		"javascript",
+		"pure",
+		"mvc",
+		"objects",
+		"microfrontend",
+		"micro-frontend",
+		"architecture",
+		"components",
+		"pure",
+		"framework",
+		"javascript-framework",
+		"mvc-pattern",
+		"demo",
+		"html",
+		"first-timers-only",
+		"microfrontends",
+		"microservices",
+		"microfrontend",
+		"component-architecture",
+		"cli",
+		"tool",
+		"nodejs",
+		"cloud",
+		"multicloud",
+		"multi-cloud",
+		"aws",
+		"server",
+		"digitalocean",
+		"hosting",
+		"architecture",
+		"n-tier",
+		"multitier",
+		"multi-tier"
+	],
+	"author": "Jean Machuca <correojean@gmail.com>",
+	"bugs": {
+		"url": "https://github.com/QuickGroup/QCObjects/issues"
+	},
+	"homepage": "https://github.com/QuickGroup/QCObjects#readme",
+	"npmName": "qcobjects",
+	"npmFileMap": [{
+		"basePath": "",
+		"files": [
+			"QCObjects.js"
+		]
+	}]
+}
```

Comment:
Looks like the formatting is inconcistent with the other package.json

Reply:
Looks to be valid, just that `npmFileMap` has the array and object openings on the same line?

Reply:
Are we not using 2 spaces instead of 4?

Reply:
Oh yeah, this is indented with tabs it looks like, should be 2 spaces, you're right.

Reply:
Ok, I've changed the indent to 2 spaces